r-python-eds-lessons/setup-r-envt.R

```diff
@@ -0,0 +1,26 @@
+##################
+# This code installs the needed R packages into the build environment
+# Author: Leah A. Wasser, Gina L. Li, Joseph V. Tuccillo
+# last modified: May 2019
```

Comment:
Since we are using version control, we don't need to include comments like this to specify who wrote what when, which quickly become out of date and/or redundant with the commit history.

Reply:
i think we could just add a contributors section to the readme to resolve this!!